libc/src/stdfix/CMakeLists.txt

```diff
@@ -46,3 +46,29 @@ foreach(suffix IN ITEMS hr r lr hk k lk uhr ur ulr uhk uk ulk)
       libc.src.__support.fixed_point.fx_bits
   )
 endforeach()
+
+add_entrypoint_object(
+  uhksqrtus
+  HDRS
+    uhksqrtus.h
+  SRCS
+    uhksqrtus.cpp
+  COMPILE_OPTIONS
+    -O3
+    -ffixed-point
```

Comment:
if `LIBC_COMPILER_HAS_FIXED_POINT` is specified, then `-ffixed-point` is automatically added to all of the compile options. You don't have to additionally specify it here.

Reply:
Removed all redundant `-ffixed-point` flags.